### PR TITLE
Delete card

### PR DESCRIPTION
**Pull request by Gw3i**

### CODE REVIEW

File: src/components/DeleteModal/DeleteModal.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import StyledButton from '../StyledButton';
+import StyledSection from '../StyledSection';
+
+export default function DeleteModal({ onDelete, videoId }) {
+	const deleteVideo = useStore(state => state.deleteVideo);
+
+	return (
+		<StyledSection>
+			<h2>Are you sure you want to delete this video?</h2>
+			<StyledButton type="button" onClick={onDelete}>
```
Comment (macfrei) [inline src/components/DeleteModal/DeleteModal.jsx]: The name `onDelete` is a bit confusing if this is the function which cancels and closes the modal. I would suggest to name it `onCancel`:
```suggestion
export default function DeleteModal({ onCancel, videoId }) {
	const deleteVideo = useStore(state => state.deleteVideo);

	return (
		<StyledSection>
			<h2>Are you sure you want to delete this video?</h2>
			<StyledButton type="button" onClick={onCancel}>
```

File: src/components/DeleteModal/DeleteModal.spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+describe('DeleteButton', () => {
+	it('should allow the delete button clicks', async () => {
+		const handleClick = jest.fn();
+		render(<StyledButton onClick={handleClick}>Delete</StyledButton>);
+		const button = screen.getByRole('button', { name: /Delete/i });
+		await userEvent.click(button);
+		expect(handleClick).toHaveBeenCalledTimes(1);
+	});
+});
```
Comment (macfrei) [inline src/components/DeleteModal/DeleteModal.spec.js]: This test does not make sense. Here you would need to mock your Zustand store again and then test that the function is called. This test only tests the button component and not the button in your modal.

File: src/components/VideoCards/VideoCards.jsx [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
 import StyledVideoContainer from '../StyledVideoContainer';
 import StyledVideoFrame from '../StyledVideoFrame';
 import StyledVideoTitle from '../StyledVideoTitle';
 
 export default function VideoCard() {
+	const [isShown, setIsShown] = useState(false);
+
+	function handleDelete() {
+		setIsShown(!isShown);
+	}
```
Comment (macfrei) [inline src/components/VideoCards/VideoCards.jsx]: `handleDelete` is not a good name for this function. When reading this, I would think that it is a function to handle your delete functionality. What it actually does is to toggle the modal. I would recommend to change the function name.

**Review by macfrei — COMMENTED** [review]
Good job, just added a few comments :)

File: src/components/DeleteModal/DeleteModal.jsx (hunk 1 above)
Comment (Gw3i) [reply src/components/DeleteModal/DeleteModal.jsx]: Done, thanks!

File: src/components/VideoCards/VideoCards.jsx (hunk 3 above)
Comment (Gw3i) [reply src/components/VideoCards/VideoCards.jsx]: Thanks, I've changed it to ```handleVisibility```

**Review by A11yship — APPROVED** [review]
👍 Looks good

**Pull request merged**